redis/utils.py

```diff
@@ -36,3 +36,41 @@ def str_if_bytes(value):
 
 def safe_str(value):
     return str(str_if_bytes(value))
+
+
+def dict_merge(*dicts):
+    """
+    Merge all provided dicts into 1 dict.
+    """
+    merged = {}
+
+    for d in dicts:
+        if not isinstance(d, dict):
+            raise ValueError('Value should be of dict type')
+        else:
+            merged.update(d)
+
+    return merged
+
+
+def list_keys_to_dict(key_list, callback):
+    return dict.fromkeys(key_list, callback)
+
+
+def merge_result(command, res):
+    """
+    Merge all items in `res` into a list.
+
+    This command is used when sending a command to multiple nodes
+    and they result from each node should be merged into a single list.
+    """
+    if not isinstance(res, dict):
```

Comment:
Same idea here. I'm not sure you should have to check the object. I'd fully support a docstring change to be specific even type hints. Any reason for this preference?

Reply:
removed